src/components/admin/AdminNewsForLauncher.vue

```diff
@@ -207,38 +202,53 @@
 import Vue from "vue";
 import { Component, Watch } from "vue-property-decorator";
 import { NewsMessage } from "@/store/admin/infoMessages/types";
-import { Editor, EditorContent, EditorMenuBar } from "tiptap";
-import { Command } from "tiptap-commands";
-import {
-  Blockquote,
-  CodeBlock,
-  HardBreak,
-  Heading,
-  HorizontalRule,
-  OrderedList,
-  BulletList,
-  ListItem,
-  TodoItem,
-  TodoList,
-  Bold,
-  Code,
-  Italic,
-  Image,
-  Link,
-  Strike,
-  Underline,
-  History,
-} from "tiptap-extensions";
 import { useOauthStore } from "@/store/oauth/store";
 import { useInfoMessagesStore } from "@/store/admin/infoMessages/store";
 import {
-  mdiCodeTags, mdiDelete,
-  mdiFileImage, mdiFormatBold, mdiFormatHeader1, mdiFormatHeader2, mdiFormatHeader3, mdiFormatItalic,
+  mdiCodeTags,
+  mdiDelete,
+  mdiFileImage,
+  mdiFormatBold,
+  mdiFormatHeader1,
+  mdiFormatHeader2,
+  mdiFormatHeader3,
+  mdiFormatItalic,
   mdiFormatListBulleted,
   mdiFormatListNumbered,
-  mdiFormatParagraph, mdiFormatQuoteClose, mdiFormatStrikethrough, mdiFormatUnderline, mdiPencil, mdiRedo, mdiUndo } from "@mdi/js";
-
-@Component({ components: { EditorContent, EditorMenuBar } })
+  mdiFormatParagraph,
+  mdiFormatQuoteClose,
+  mdiFormatStrikethrough,
+  mdiFormatUnderline,
+  mdiMinus,
+  mdiPencil,
+  mdiRedo,
+  mdiUndo,
+} from "@mdi/js";
+import { Editor, EditorContent } from "@tiptap/vue-2";
+import { Document } from "@tiptap/extension-document";
+import { Paragraph } from "@tiptap/extension-paragraph";
+import { Text } from "@tiptap/extension-text";
+import { History } from "@tiptap/extension-history";
+import { Strike } from "@tiptap/extension-strike";
+import { Underline } from "@tiptap/extension-underline";
+import { Italic } from "@tiptap/extension-italic";
+import { Bold } from "@tiptap/extension-bold";
+import { Code } from "@tiptap/extension-code";
+import { Link } from "@tiptap/extension-link";
+import { Blockquote } from "@tiptap/extension-blockquote";
+import { Image } from "@tiptap/extension-image";
+import { BulletList } from "@tiptap/extension-bullet-list";
+import { ListItem } from "@tiptap/extension-list-item";
+import { Heading } from "@tiptap/extension-heading";
+import { HardBreak } from "@tiptap/extension-hard-break";
+import { CodeBlock } from "@tiptap/extension-code-block";
+import { HorizontalRule } from "@tiptap/extension-horizontal-rule";
+import { OrderedList } from "@tiptap/extension-ordered-list";
+import { TaskList } from "@tiptap/extension-task-list";
```

Comment:
Are we using TaskList? Do we need the import? Same question goes for the rest of the imports.

Reply:
I don't see it in the menubar, hence my question :)

Reply:
Don't worry about it if you don't know. We can fix it later, too.

Reply:
They renamed `Todo*` to `Task*`
It was used before here:
https://github.com/w3champions/website/pull/680/files#diff-eda8fafd78e785a9c6b4b2873a29cac23ec7186e729933ca071f1799cdbfe48cL221-L222
But not sure if it was really used 🤔

Reply:
Could we import only the extensions we use? Or we can remove them later, but might as well do it now!